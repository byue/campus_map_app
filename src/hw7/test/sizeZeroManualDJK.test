# Simple test of Djikstra's algorithm.
# Verifies output of DJK for size 0 graph.

CreateGraph g

FindPath g a b