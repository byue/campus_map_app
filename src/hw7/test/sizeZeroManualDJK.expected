# Simple test of Djikstra's algorithm.
# Verifies output of DJK for size 0 graph.

created graph g

unknown character a
unknown character b